Document: add composite version columns (#686)
risingwavelabs/risingwave-docs: sql/commands/sql-create-table.mdx

    automatically compact "no-op" updates to prevent massive updates for downstream jobs. You can create a logical view with `SELECT * EXCEPT(version_col)` to avoid depending on the version column by mistake.
</Note>

> **Multiple version columns**

Reviewer: ```suggestion
**Composite version columns**
```

<Note>
Added in v2.6.0.
</Note>

> RisingWave supports specifying multiple version columns to enable more sophisticated conflict-resolution strategies for CDC and upsert scenarios.

Reviewer: ```suggestion
RisingWave supports specifying composite version columns in the `VERSION COLUMN` clause.
```
